Sheet 2/ProblemF.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
```

Comment:
Your solution is great 👍 

For your information, there's another solution that doesn't need loops which use something called "bit-wise operations" in C++. A solution would thus be `a ^ b`. (But that doesn't reduce the value of your solution at all)